<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>채식주의 선택</string>
    </property>
    <widget class="QComboBox" name="vegCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>해당없음</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>비건</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>락토 베지테리언</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>오보 베지테리언</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>락토 오보 베지테리언</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>페스코 베지테리언</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>폴로 베지테리언</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>기본 정보 입력</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setPass_text"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>271</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>성별</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="maleBtn">
        <property name="text">
         <string>남</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="femaleBtn">
        <property name="text">
         <string>여</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>나이</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="getAge_box"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>271</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>아이디</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setID_text"/>
      </item>
      <item>
       <widget class="QPushButton" name="checkOverlapBtn">
        <property name="text">
         <string>중복확인</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="alle_groupbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>식품 알레르기 종류 선택 (중복 가능)</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>160</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QCheckBox" name="peanut_check">
        <property name="text">
         <string>땅콩</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="milk_check">
        <property name="text">
         <string>우유</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="bean_check">
        <property name="text">
         <string>대두</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="wheat_check">
        <property name="text">
         <string>밀</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="egg_check">
        <property name="text">
         <string>난류</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="buckwheat_check">
        <property name="text">
         <string>메밀</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="beef_check">
        <property name="text">
         <string>쇠고기</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>160</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="shrimp_check">
        <property name="text">
         <string>새우</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="crab_check">
        <property name="text">
         <string>게</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="peach_check">
        <property name="text">
         <string>복숭아</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="pork_check">
        <property name="text">
         <string>돼지고기</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="mackerel_check">
        <property name="text">
         <string>고등어</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="tomato_check">
        <property name="text">
         <string>토마토</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="squid_check">
        <property name="text">
         <string>오징어</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>81</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QCheckBox" name="walnut_check">
        <property name="text">
         <string>호두</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="kiwi_check">
        <property name="text">
         <string>키위</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="acid_check">
        <property name="text">
         <string>아황산류</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="pinenut_check">
        <property name="text">
         <string>잣</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chicken_check">
        <property name="text">
         <string>닭고기</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="calm_check">
        <property name="text">
         <string>조개류</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="sesame_check">
        <property name="text">
         <string>참깨</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="none_check">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>해당 없음</string>
     </property>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="signinBtnBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
